CodingStyle

```diff
@@ -32,8 +32,9 @@ by section.
       Yes, _t also means typedef.  It's perhaps not ideal.
 
         struct my_type_t {
-          int a, b;
-          my_type_t() : a(0), b(0) {}
+          int a = 0, b = 0;
+	  void encode(...) ...
```

Comment:
I think I understand: the recommendation for _t has nothing to do with the presence of member functions (ie. it doesn't need to be what until-recently-was-called Plain Old Data (POD)), but rather the /intent/ of the type. Cool, I think the example helps clarify the rule.